--- a/Sweet-Dark/kde/colorschemes/Sweet.colors
+++ b/Sweet-Dark/kde/colorschemes/Sweet.colors
@@ -97,9 +97,9 @@
 contrast=4
 
 [WM]
-activeBackground=47,52,63
-activeBlend=47,52,63
-activeForeground=211,218,227
-inactiveBackground=47,52,63
-inactiveBlend=47,52,63
-inactiveForeground=102,106,115
+activeBackground=12,14,20
+activeBlend=12,14,20
+activeForeground=202,203,209
+inactiveBackground=12,14,20
+inactiveBlend=12,14,20
+inactiveForeground=115,117,122

